### PR TITLE
S99.26 Add salesforce.

### PR DESCRIPTION
**Pull request by swordbeta**

### CODE REVIEW

**Comment by swordbeta** [comment]
test this please

**Comment by swordbeta** [comment]
test this please

**Comment by pvaass** [comment]
test this please

File: Lib/Salesforce/SalesforceAPI.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		])->getToken();
+	}
+
+	/**
+	 * @param $name
+	 * @param $arguments
+	 *
+	 * @return mixed
+	 */
+	public function __call($name, $arguments) {
```
Comment (pvaass) [inline Lib/Salesforce/SalesforceAPI.php]: Smeerlap

**Pull request merged**